universal-application-tool-0.0.1/project/Webpack.scala

```diff
@@ -0,0 +1,31 @@
+import play.sbt.PlayRunHook
```

Comment:
question: Does this have to be configured in scala?  I don't really know scala.

Reply:
Might be a good question for James Ward

Reply:
I honestly have no idea. I got this entire setup from the answer on [this stackoverflow post](https://stackoverflow.com/q/34568008) (and [this github repo](https://github.com/nouhoum/play-react-webpack) linked from there).

Definitely down to eliminate this lone `.scala` file if we can though.

Reply:
@bion Do you think I can add James as a reviewer to this PR? What's the best way to connect with them?

Reply:
According to James, since we're changing the sbt build, it must be in Scala.

Of course, if we move forward with #235 we could get rid of this altogether.

Reply:
Thanks for getting in touch!